.github/workflows/generated_cmake.yml

```diff
@@ -1,8 +1,10 @@
-name: Generated CMake File Lists
+name: Generate CMake File Lists
 
 on:
-  - push
-  - pull_request
+  push:
+    branches:
+      - main
+      - '[0-9]+.x'
```

Comment:
This won't work on 21.x will it?

Reply:
Right, I don't think this will work on 21.x. I was assuming that it would automatically be disabled on that branch since this code will never make it onto 21.x, but now that I think of it I'm not entirely sure how GitHub handles this, so I went ahead and excluded 21.x explicitly.

Reply:
Ah right, it probably wouldn't affect 21.x anyway (the original workflow also would have been broken)